### ISSUE TITLE
Lesson Api Redesign

### ISSUE THREAD

**Issue by Jacksunwei** [genfengxue/gale#1]
Redesign the lesson API, the following should be taken into consideration. 
- the uri of course video
- the api should accept the courseId as parameter

**Comment by lutaoact** [comment]
@Jacksunwei 

call the api like this:
http://data.genfengxue.com/api/lessons?courseNo=2

courseNo=2 indicates the course is "direct english"

There are 81 lessons in this course. You can test now.

video uri will be integrated later.

**Comment by Jacksunwei** [comment]
You can submit code with keywords like fix (for bug), resolve (for new feature), plus #[issue_num], to relate your commit with certain issue. Then the issue will be closed automatically. 
see [this](https://help.github.com/articles/closing-issues-via-commit-messages/)

**Comment by lutaoact** [comment]
I will use commit message "closes #1" after I integrate the video url in lesson model. Thanks for your suggestions.

**Comment by lutaoact** [comment]
data structure for lesson

``` coffeescript
      courseNo:
        type: Number
        required: true
      lessonNo:
        type: Number
        required: true
      englishTitle:
        type: String
        required: true
      chineseTitle:
        type: String
        required: true
      imageUrl:
        type: String
      videoUrl:
        type: String
```
